examples/GTest/foo.cpp

```diff
@@ -1,5 +0,0 @@
-#include <gtest/gtest.h>
```

Comment:
Please leave this example untouched. This is a quite common pattern to use GTest and it need to be tested. Add separate example for other cases, like `examples/GTest-no-main`, `examples/GMock`.